doc/man/man7/elektra-backends.7

```diff
@@ -16,7 +16,7 @@ It was impossible to implement powerful and feature\-rich backends so far becaus
 .P
 To solve this dilemma, Elektra uses \fBmultiple plugins\fR together to build up a backend\. The key set processed by one plugin will be passed to the next\. This approach allows us to implement separate features in separate plugins\. Plugins concerned with reading from and writing to permanent storage are called \fBstorage plugins\fR\. We cleaned existing backends from other tasks so that their only job now is related to the storage\. Using this approach, the plugins provide the desired separation of concerns inside a backend\.
 .P
-Each plugin implements a single concrete requirement and it does that well\. This architecture allows plugins to have external dependence\. Not every plugin has the burden to be portable anymore\. That is no problem because the plugins are separate subprojects and maintainers can decide if they should be built for a specific platform or not\. Afterwards users can choose which plugins they want to install and use\. And finally, the administrator can choose which of the plugins should be loaded for each backend\. If a specific feature is not needed, it is not included and does not cause additional overhead\. Given the chosen approach, the core of Elektra can stay minimal\. Most functionality can be moved to plugins\.
+Each plugin implements a single concrete requirement and it does that well\. This architecture allows plugins to have external dependencies\. Not every plugin has the burden to be portable anymore\. That is no problem because the plugins are separate subprojects and maintainers can decide if they should be built for a specific platform or not\. Afterwards users can choose which plugins they want to install and use\. And finally, the administrator can choose which of the plugins should be loaded for each backend\. If a specific feature is not needed, it is not included and does not cause additional overhead\. Given the chosen approach, the core of Elektra can stay minimal\. Most functionality can be moved to plugins\.
```

Comment:
```suggestion
Each plugin implements a single concrete requirement and it does that well\. This architecture allows plugins to have external dependencies\. Not every plugin has the burden to be portable anymore\. That is no problem because the plugins are separate subprojects and maintainers can decide if they should be built for a specific platform or not\. Afterward, users can choose which plugins they want to install and use\. And finally, the administrator can choose which of the plugins should be loaded for each backend\. If a specific feature is not needed, it is not included and does not cause additional overhead\. Given the chosen approach, the core of Elektra can stay minimal\. Most functionality can be moved to plugins\.
```

Reply:
Found this additional typo in the original file. It is easier to fix this here than create another pr.